lib/Utils.js

```diff
@@ -0,0 +1,49 @@
+const REPEAT_MARKER = [0x69, 0x72];
+
+module.exports = class LinuxDeviceUtils {
+  /**
+   * Decodes the custom buffer that includes the interval and repetitions
+   *
+   * @returns {*}
+   * @param chunk
+   */
+  static decodeWriteBuffer(chunk) {
+    // Custom parsing for IR header.
+    // The stream.write function only accepts an buffer, therefore we need to encode the interval and repetitions in the buffer before using write().
+    let repetitions = 1;
```

Comment:
Should the default be 1? This means that every command passed through `_writePromise` will get a repetition set to 1. For example also Zigbee commands. Also, does repetition=1 mean the signal is broadcasted twice?